src/backend/templates/event.hbs

```diff
@@ -32,10 +32,11 @@
     </head>
 
     <body onload="datafetcher()">
+      <a href="#content" class="skip">Skip to content</a>
     {{> scroll}}
     <a id="top"></a>
     {{> navbar}}
-    <div class="carousel slide" style="margin-bottom: 0;">
+    <div class="carousel slide" style="margin-bottom: 0;" id="content">
```

Comment:
@anu-007 That make sense `<a>` element take its id and matches with all id on the page to scroll.